### DISCUSSION TITLE
Library of voices and voice cloning support

### DISCUSSION THREAD

**Discussion by robertkirathe** [souzatharsis/podcastfy#193] [Q&A]
Hi, 

Does anyone know if podcastify has a library of different voices that we can use, or it is just the two voices that are in the sample audios generated? Also, does it support voice cloning - can I use my own voice to generate a podcast?

**Comment by souzatharsis** [comment]
https://github.com/souzatharsis/podcastfy/blob/main/usage/how-to.md#how-to-use-your-own-voice-in-audio-podcasts

**Comment by Cfomodz** [comment]
You can use your own voice by creating a Voice Clone (or PVC (professional voice clone)) through [Elevenlabs](https://try.elevenlabs.io/api). These clones can also be used through Eleven Reader.
